app/src/main/java/com/example/cs4520_twitter/ProfileScreen.kt

```diff
@@ -0,0 +1,385 @@
+            ) {
+                Text(
+                    text = "Edit",
+                    fontSize = 14.sp
+                )
+            }
+            // Button for viewing my followers
+            Button(
+                onClick = {},
+                modifier = Modifier
+                    .height(35.dp)
+                    .width(110.dp)
+                    .constrainAs(myFollowersButton) {
+                        top.linkTo(userBanner.top, margin = 10.dp)
+                        absoluteRight.linkTo(
+                            userBanner.absoluteRight,
+                            margin = 10.dp
+                        )
+                    },
+                contentPadding = PaddingValues(0.dp),
+                border = BorderStroke(2.dp, blue),
+                colors = ButtonDefaults.buttonColors(contentColor = blue, containerColor = Color.White)
+            ) {
+                Text(
+                    text = "My Followers",
+                    fontSize = 14.sp
+                )
+            }
+            // user desc.
+            TextField(
+                value = userDescText,
+                onValueChange = { userDescText = it },
+                enabled = false, // uncomment this to allow users to type
+                singleLine = true,
+                colors = TextFieldDefaults.textFieldColors(
+                    containerColor = Color.White,
+                    focusedIndicatorColor =  Color.Transparent,
+                    unfocusedIndicatorColor = Color.Transparent,
+                    disabledIndicatorColor = Color.Transparent),
+                modifier = Modifier
+                    .height(60.dp)
+                    .width(350.dp)
+                    .constrainAs(userDesc) {
+                        top.linkTo(username.bottom, margin = 0.dp)
+                        absoluteRight.linkTo(
+                            userBanner.absoluteRight,
+                            margin = 25.dp
+                        )
+                    })
+
+            // List of Babs
+            LazyColumn(userScrollEnabled = true,
+                modifier = Modifier
+                    .height((maxHeight * 0.5).dp) // half the screen
+                    .width((maxWidth * 0.93).dp)  // most of the screen width
+                    .constrainAs(babColumn) {
+                        top.linkTo(userBanner.bottom, margin = 10.dp)
+                        absoluteLeft.linkTo(parent.absoluteLeft, margin = ((maxWidth * 0.07)/2).dp)
+                    }) {
+                items(
+                    count = dummyBabList.size,
+                    itemContent = { index ->
+                        BabCard(dummyBabList[index])
+                    }
+                )
+            }
+        }
+    }
+}
+
+// Bab card for Bab list
+@OptIn(ExperimentalGlideComposeApi::class)
+@Composable
+fun BabCard(bab : Bab) {
```

Comment:
This looks great! Can the BabCard be moved to a separate file so it can more easily be used across screens?